GVFS/GVFS.Installers/GVFS.Installers.csproj

```diff
@@ -28,9 +28,7 @@
     <Content Include="$(PkgGitForWindows_GVFS_Portable)\tools\*" />
     <Content Include="install.bat" />
     <Content Include="info.bat" />
-    <NuspecTemplate Include="GVFS.Installers.template.nuspec"

```

Comment:
Please can you revert the other changes to this file (not the `Tools.InnoSetup` version bump of course)?

Reply:
Done